README.md

```diff
@@ -1,8 +1,99 @@
-In order to build the API to access the data correctly, you'll need to:
-
-- Set up two .env files and set the database name in the global environment.
-- Install the relevant DevDependencies and dependencies on your machine.
-- Run the setup and seed (you can find the scripts in the package.json) to create the databases and seed the data.
-- You will be interacting with the PSQL database through node-postgres when fetching the data.
-- There are two sets of data - test and dev but you can use the index.js file to require in the relevant sets.
-- The connection.js file will automatically determine which environment will run and will return an error if it doesn't find the database.
+# NC News
+
+A backend project for Northcoders News, a reddit-like app used to access a database of articles and comments.
+
+A link to the project can be found here: https://nc-news-backend-yyld.onrender.com/api
+
+## Getting Started
+
+### Pre-requisites:
+
+Before you start, ensure you have installed the following:
+
+- Postgres
+
+  To install Postgres, go to: [Postgres](https://www.postgresql.org/download/)
+
+  The minimum version required is v14.10
+
+- Node.js
+
+  To install Node, go to: [Node](https://nodejs.org/en/download/)
+
+  The minimum version required is v21.4.0
+
+### Installation:
+
+Follow these instructions to help you run a copy of the project on your local machine.
+
+1. Clone a copy of the repository on to your local machine using the command below:
+
+   ` git clone https://github.com/kimberley-lee/nc-news-backend.git`
+
+2. Install the associated devDependencies and dependencies:
+
+   `npm install`
+
+### Setting up `.env` files:
+
+In order to create your database and seed the relevant data, you will need to:
+
+1. Set up three '.env' files in the root and name them:
+
+   `.env.test`
+
+   `.env.development`
+
+   `.env.production`
+
+2. set the database name in the global environment:
+
+   In the '.env.test', type in:
+
+   `PGDATABASE=nc_news_test`
+
+   In the '.env.development', type in:
+
+   `PGDATABASE=nc_news`
+
+   In the '.env.production', type in:
+
+   `DATABASE_URL=postgres://fkfdizrn:mYJRbNlDGLTWM2A5n8zfN9858BX19NiS@flora.db.elephantsql.com/fkfdizrn`
```

Comment:
delete this!! this isn't safe to publicise (and the whole reason we hide them using .env files)